### DISCUSSION TITLE
I can not find the parent repository

### DISCUSSION THREAD

**Discussion by nikolayromenskiy** [community/community#141476] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

My https://github.com/nikolayromenskiy/tmatic is forked from https://github.com/evgrmn/tmatic, but /evgrmn/tmatic shows error 404 now. It was available just this day, but now seems to be deleted. Neither me nor the /evgrmn/tmatic owner has no idea what's happened. Please anybody give me a hint what could have happened.

**Comment by davevad93** [comment]
Hi @nikolayromenskiy ,

The [evgrmn](https://github.com/evgrmn) profile returns a 404 error, which means it's no longer on `GitHub`, perhaps it has been deleted.

**Comment by nikolayromenskiy** [comment]
A year of work, more than 1000 commits. How can find a reason why it was deleted and by whom?

**Reply by davevad93** [reply]
I don't know @nikolayromenskiy , [evgrmn](https://github.com/evgrmn) seems to be no longer present on `GitHub`, maybe the owner himself has deleted the profile.

**Comment by nikolayromenskiy** [comment]
No, the owner hasn't. The account has been flagged by Github. The reason is unknown. The owner appealed yesterday. No answer from support yet.

**Reply by davevad93** [reply, marked as answer]
I see @nikolayromenskiy , well support can take a while to respond because they have to deal with a lot of daily requests, it can take even weeks, so the owner has to be patient while waiting to get a reply. I I hope everything will be solved.

**Reply by nikolayromenskiy** [reply]
It's a shame that honest work can be turned into nothing in one moment with no reason. Thank you for your regard to the problem.
